front/ticket.form.php

```diff
@@ -203,6 +203,12 @@
 } else {
    Html::header(__('New ticket'),'',"helpdesk","ticket");
    unset($_REQUEST['id']);
+   if (isset($_GET['id'])) {
```

Comment:
isset statements for GET and POST are superflu, unset do nothing (and did no warnings) if the var is not set